packages/shorebird_code_push_protocol/lib/src/messages/create_patch_artifact/create_patch_artifact_request.dart

```diff
@@ -32,6 +33,9 @@ class CreatePatchArtifactRequest {
   /// The hash of the artifact.
   final String hash;
 
+  /// The signature of the [hash].
+  final String? hashSignature;
```

Comment:
Might be nice to explain why this field is nullable and what happens if it's null vs not null in the doc comment above.